### DISCUSSION TITLE
Place command failed

### DISCUSSION THREAD

**Discussion by suppamax** [siliconcompiler/siliconcompiler#915] [Help (Q&A)] [answered]
While testing siliconcompiler on one pf my projects, I get this error
```
| WARNING | job0    | place        | 0   | Command failed. See log file /Workarea/sc/build/btc_miner/job0/place/0/place.log
```
[place.log](https://github.com/siliconcompiler/siliconcompiler/files/8217179/place.log)

Could you please help me understand the problem?

**Comment by nmoroze** [comment, marked as answer]
Interesting, those seem like normal logs that have just been cut off suddenly. Is there any chance your machine ran out of memory during the run?
